x-pack/plugins/security/public/management/users/components/change_password_form/change_password_form.tsx

```diff
@@ -118,6 +120,7 @@ export class ChangePasswordForm extends Component<Props, State> {
           }
         >
           <EuiFieldText
+            autoComplete="off"
```

Comment:
```suggestion
            autocomplete="new-password"
```
We should treat this as a "new password" field, so browsers can react accordingly.